tests/integration/run-as-production-env.js

```diff
@@ -32,16 +32,37 @@ function spawnWithShell(cmd, args, options) {
   return spawn(cmd, args, Object.assign({}, baseSpawnOptions, options));
 }
 
+function moveFile(oldPath, newPath) {
+  return new Promise((resolve, reject) => {
+    fs.rename(oldPath, newPath, (err) => (err ? reject(err) : resolve()));
+  });
+}
+
+function getPackedName() {
+  return new Promise((resolve, reject) => {
+    fs.readFile('package.json', (err, data) => {
+      if (err) {
+        reject(err);
+      } else {
+        const info = JSON.parse(data.toString());
+        resolve(`${info.name}-${info.version}.tgz`);
+      }
+    });
+  });
+}
+
 function createPackage(archiveFilePath) {
   console.log(chalk.green('Create a pre-release npm package archive'));
   return new Promise((resolve, reject) => {
-    const pkgPack = spawnWithShell('yarn', ['pack', '--filename', archiveFilePath]);
+    const pkgPack = spawnWithShell('npm', ['pack']);
 
     pkgPack.stdout.pipe(process.stdout);
     pkgPack.stderr.pipe(process.stderr);
     pkgPack.on('close', (exitCode) => {
       if (exitCode === 0) {
-        resolve();
+        resolve(getPackedName().then((filename) => {
+          return moveFile(filename, archiveFilePath);
```

Comment:
Instead of renaming, the file I am wondering whether it wouldn't be easier to just use the right generated package name in https://github.com/mozilla/addons-linter/blob/c71099f3c05dd6b3bf0924b794b29c657e9dc6da/tests/integration/run-as-production-env.js#L108

It might still involve getting the package name and version from the manifest there, but we would save the renaming.